hack/local-up-cluster.sh

```diff
@@ -253,7 +253,22 @@ function start_controller_manager {
 
 function start_kubelet {
     KUBELET_LOG=/tmp/kubelet.log
+
     if [[ -z "${DOCKERIZE_KUBELET}" ]]; then
+      mkdir -p /var/lib/kubelet
```

Comment:
Actually...

You can keep this here, and then on line 295: change this:

```
 --volume=/var/lib/kubelet/:/var/lib/kubelet:rw \
```

to:

` --volume=/var/lib/kubelet/:/var/lib/kubelet:rwZ \`

...and docker will relabel the volume if selinux is enabled in docker.